semgrep/semgrep/target_manager.py

```diff
@@ -96,7 +96,7 @@ def converted_pipe_targets(targets: Sequence[str]) -> Iterator[Sequence[str]]:
 
 
 @define
-class IgnoreLog:
+class IgnoreOrFailedLog:
```

Comment:
```suggestion
class Log:
```
or
```suggestion
class TargetingLog:
```
?

Reply:
What's your thinking here @underyx? I'd prefer more verbose, but I guess `TargetingLog` is just more generic? Maybe `FileTargetingLog`?